#Config file extracted and adapted from (https://github.com/lorenwel/linefit_ground_segmentation)

n_threads: 4                # number of threads to use.

r_min: 0.5                  # minimum point distance.
r_max: 50                   # maximum point distance.
n_bins: 120                 # number of radial bins.
n_segments: 360             # number of radial segments.

max_dist_to_line: 0.25      # maximum vertical distance of point to line to be considered ground.

sensor_height: 2.0          # sensor height above ground.
min_slope: -0.4              # minimum slope of a ground line.
max_slope: 0.4              # maximum slope of a ground line.
max_fit_error: 0.25 #0.15         # maximum error of a point during line fit.
long_threshold: 2.5         # distance between points after which they are considered far from each other.
max_long_height: 0.25 #0.15        # maximum height change to previous point in long line.
max_start_height: 0.25 #0.15       # maximum difference to estimated ground height to start a new line.
line_search_angle: 0.1      # how far to search in angular direction to find a line [rad].

gravity_aligned_frame: ""  # Not used since sweeps are gravity aligned

latch: false               # latch output topics or not
visualize: false           # visualize segmentation result - USE ONLY FOR DEBUGGING

